frontend/src/metabase/css/core/colors.module.css

```diff
@@ -49,7 +49,7 @@
   --mb-color-bg-black: #2e353b;
   --mb-color-bg-dark: #93a1ab;
   --mb-color-bg-medium: #edf2f5;
-  --mb-color-bg-light: #f9fbfc;
+  --mb-color-bg-light: var(--mb-base-color-gray-10);
```

Comment:
this is a significant color change that needs design review - see

Before | after
---|---
![Screen Shot 2024-09-02 at 9 31 00 AM](https://github.com/user-attachments/assets/2842fc60-a9af-4760-aa18-9f24c40d93c8) | ![Screen Shot 2024-09-02 at 9 30 55 AM](https://github.com/user-attachments/assets/e5d88945-7705-40ea-bf0c-77c7ea9cba0c)

Reply:
I'm using the new color map [from here.](https://www.figma.com/design/OlyKP09vro7EkQ2ZzXDDla/color-playground?node-id=501-2549&node-type=FRAME&t=xzjsTwjJ4FolPMur-0)
![Screenshot 2024-09-03 at 1 49 16 PM](https://github.com/user-attachments/assets/9c86571f-c55c-4364-aee4-6f033951f81d)

Is this up for implementation yet @AlessioLaiso @kdoh, if not I can hold these changes off. But if we have the palette ready, I think we should at least start using it to see what impact this palette brings to the app.

Reply:
Updated the color now using the [latest mapping.](https://www.figma.com/design/YxiDXi1DzmXJCTMCuqXaaY/Applying-the-updated-colors?node-id=4001-16486&node-type=FRAME&t=1YHSpmVzKFQG2p8u-0) This should make this color looks similar to the previous one.
![image](https://github.com/user-attachments/assets/e47a7797-60a9-4aa8-8b96-d435c3028797)

Reply:
ultimately up to design, but I'd rather see major color changes in an isolated PR

Reply:
That's fair, if our designers think we should revert them then I will. Now I just got a confirmation from Alessio that [it's fine to use the new colors.](https://github.com/metabase/metabase/pull/47189#issuecomment-2327039777)